README.md

```diff
@@ -84,10 +86,15 @@ Supported Cars
 | Honda     | Pilot 2019                    | All               | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Honda     | Ridgeline 2017-19             | Honda Sensing     | openpilot        | 25mph<sup>5</sup>  | 12mph        |
 | Hyundai   | Elantra 2017-19<sup>1</sup>   | SCC + LKAS        | Stock            | 19mph              | 34mph        |
+| Hyundai   | Elantra GT 2017-19<sup>1</sup>| All               | Stock            | 0mph               | 30mph        |
```

Comment:
Base GT in the USA doesn't have SCC, requires Tech Package. I think all have LKAS. Probably should be changed to `SCC + LKAS`

Reply:
merge